pom.xml

```diff
@@ -19,7 +19,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <java.plugin.version>5.14.0.18788</java.plugin.version>
-    <sslr.version>1.23</sslr.version>
+    <sslr.version>1.21</sslr.version>
```

Comment:
So we downgraded a dependency here? If that was required and on purpose I assume this is OK. I do not even remeber what `sslr` was. I am just commenting as typically we upgrade versions and you should double-check that this is intended.

Reply:
[Commit from 08.11.](https://github.com/lmarniazman/sonar-devon4j-plugin/commit/dd0cf0f234ffbc25068a929fca5f4112e25ae6c6) (Upgrade)
[Commit from 13.12.](https://github.com/lmarniazman/sonar-devon4j-plugin/commit/dad0e6bc5a3dfcecf745d9fe58abeba31d07fe77) (Downgrade)

It seems that I must have resolved a merge conflict somewhere the wrong way. I compared the poms of our master and of this PR branch and I don't really see any other versions that don't match. Good catch though, I will correct it.